views/add.handlebars

```diff
@@ -1,2 +1,7 @@
 //directed to from the home page 'add' button		
-//uses partial form.hbs 
\ No newline at end of file
+//uses partial form.hbs 
+
+<form action="">
+  {{<form}}
+  <input> add topic </input>
```

Comment:
HTML inputs don't usually use a closing tag

you can just use: `<input type='submit' value='Add topic'>`